test/tests/interfaces/pbs_preempt_params.py

```diff
@@ -325,7 +310,7 @@ def test_set_unset_preempt_sort(self):
         else:
             self.fail(msg)
```

Comment:
I think we can remove line 283 to 312 as this we already doing in line 281: self.common_tests(param, msg).

Reply:
right - will remove these lines.